model/token_service.go

```diff
@@ -49,7 +49,7 @@ type Token interface {
 	Payload() map[string]string
 }
 
-//Validator calidate token with external requester
+//Validator validate token with external requester
```

Comment:
Please add one space after dashes. Also, validate**s**. Also, the period at the end of a sentence is needed.